.travis.yml

```diff
@@ -59,6 +35,7 @@ before_install:
       sudo apt-get update
       sudo apt-get install -y libc6:i386 libncurses5:i386 libstdc++6:i386
       sudo apt-get install -y lcov
+      sudo apt-get install -y libmatio2 libmatio-dev
```

Comment:
if you install both you will not know if you used the static or the dynamic lib

Reply:
We are not linking statically against matio. Only dynamic. however, we do need the headers.

Reply:
ldd should tell us. We really need to find a manner to test this